lib/algoliasearch/tasks/algoliasearch.rake

```diff
@@ -1,9 +1,14 @@
 namespace :algoliasearch do
- 
+
   desc "Reindex all models"
   task :reindex => :environment do
     AlgoliaSearch::Utilities.reindex_all_models
   end
+
+  desc "Set settings to all indexes"
+  task :set_settings => :environment do
```

Comment:
rename to set_all_settings or at least add option to pass model to push only one Model